content/en/docs/contribute/new-content/new-features.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Documenting a new feature
+linktitle: New features
+content_template: templates/concept
+main_menu: true
+weight: 20
+card:
+  name: contribute
+  weight: 45
+  title: Documenting a new feature
+---
+{{% capture overview %}}
+
+Each major Kubernetes release includes new features.
+Most of these features need documentation to show people how to use them.
+
+Generally, the SIG responsible for a feature submits draft documentation for the
+feature as a pull request to the appropriate release branch of
+`kubernetes/website` repository, and someone on the SIG Docs team provides
+editorial feedback or edits the draft directly. This section covers the branching
+conventions and process used by both groups.
```

Comment:
processes during release time or in general?

Reply:
Excellent question – this is mostly inherited content, so I'm going to assume it's release time processes.